templates/2.0/mapping.json

```diff
@@ -342,7 +342,7 @@
             "require_allow_list_signatures": {
                 "section": "cloud_verifier",
                 "option": "require_allow_list_signatures",
-                "default": "True"
+                "default": "False"
```

Comment:
do we need to worry about changing the old templates default?

Reply:
I don't want a True to ever appear in `require_allow_list_signatures` because it would change the behavior of keylime_tenant tool requiring policies to be signed.